tst/testinstall/syntaxtree.tst

```diff
@@ -2854,6 +2861,7 @@ rec(
   stats := rec(
       statements := [ rec(
               obj := rec(
+                  string := "1.0",
                   type := "T_FLOAT_EXPR_EAGER",
                   value := 1 ),
```

Comment:
I am confused as to why `value` is 1 here... I'd expect `1.` or `1.0`... huh

Reply:
This is part of the other PR #114 and should maybe be discussed there.